okhttp/src/test/java/okhttp3/CallTest.java

```diff
@@ -939,20 +941,26 @@ private void postBodyRetransmittedAfterAuthorizationFail(String body) throws Exc
   }
 
   /** https://github.com/square/okhttp/issues/4761 */
-  @Test public void interceptorCallsProceedWithoutClosingPriorResponse() throws Exception {
+  @Test
+  public void interceptorCallsProceedAndClosesPriorResponse() throws Exception {
     server.enqueue(new MockResponse()
+        .setBodyDelay(250, TimeUnit.MILLISECONDS)
```

Comment:
This avoids flakiness, e.g. response must not complete normally before the test otherwise finishes.